hybrasyl/Objects/ItemObject.cs

```diff
@@ -256,7 +256,7 @@ public void Invoke(User trigger)
                     return;
                 }
 
-                if (!invokeScript.ExecuteFunction("OnUse", this, trigger, this))
+                if (!invokeScript.ExecuteFunction("OnUse", trigger, this, this))
```

Comment:
`public bool ExecuteFunction(string functionName, dynamic invoker, dynamic source, dynamic scriptItem=null)`
for `ItemObject` probably better to do `trigger, null, this`. this is a good catch